code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1891,7 +1891,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeysdelight/New()
 	..()
-	reagents.add_reagent(NUTRIMENT, 10)
+	reagents.add_reagent(NUTRIMENT, 15)
```

Comment:
It's actually a pretty decent amount. Monkey's Delight inherits the 10 nutriment from a monkeycube to become 20 nutriment (double the amount of nutriment in the 3 pieces of meat you get from butchering a monkey, which is 9). This buff is raising that to 25.

I'm not sure 4noraisins should be the standard for anything. 6 nutriment is 90 nutrition, e.g.: fairly hearty.